### PR TITLE
For PR #808, Patch updates for docs , mainly for layout

### PR DESCRIPTION
**Pull request by rdlrt**

### CODE REVIEW

File: docs/docker/run.md [hunk 1]
```diff
@@ -1,41 +1,39 @@
...
 ```bash
-docker run -dit 
--e NETWORK=mainnet 
---name <YourCName>
--p <your_custom_path>:/opt/cardano/cnode/priv
--p <your_custom_db_path>:/opt/cardano/cnode/db
+docker run -dit \
+-e NETWORK=mainnet \
+--name <YourCName> \
+-p <your_custom_path>:/opt/cardano/cnode/priv \
```
Comment (TrevorBenson) [inline docs/docker/run.md:17]: @rdlrt @redoracle the -p flag is for exposing ports. The -v flag is for binding a docker volume or external path. I believe line 17 & 18 should instead be:
```
-v <your_custom_path>:/opt/cardano/cnode/priv \
-v <your_custom_db_path>:/opt/cardano/cnode/db \
```

File: docs/docker/run.md [hunk 2]
```diff
@@ -1,41 +1,39 @@
...
--e NETWORK=mainnet  
--p <your_custom_path>:/opt/cardano/cnode/priv
--p <your_custom_db_path>:/opt/cardano/cnode/db
-cardanocommunity/cardano-node 
+docker run -dit \ 
+--name <YourCName> \
+-e NETWORK=mainnet \
+-p 6000:6000 \
+-e NETWORK=mainnet \
+-p <your_custom_path>:/opt/cardano/cnode/priv \
```
Comment (TrevorBenson) [inline docs/docker/run.md:34]: Same here, above it properly uses `-p 6000:6000` on line 32, but then re-uses -p instead of -v on line 34 and 35.

**Review by TrevorBenson — CHANGES_REQUESTED** [review]
A few things to fix, but once we have correct flags for volume the rest LGTM.

File: docs/docker/run.md (hunk 1 above)
Comment (rdlrt) [reply docs/docker/run.md]: Resolved on base PR

File: docs/docker/run.md (hunk 2 above)
Comment (rdlrt) [reply docs/docker/run.md]: Resolved on base PR

**Pull request merged**